src/interactor/MemberInteractor.java

```diff
@@ -0,0 +1,23 @@
+package interactor;
+
+import boundary.IMember;
+import entity.MemberTask;
+import entity.Profile;
+import gateway.IGateway;
+
+public class MemberInteractor implements IMember {
+    public IGateway gateway;
+
+    public MemberInteractor(IGateway g){
+        gateway = g;
+    }
+
+    public Profile viewProfile(String email){
+        Profile profile = gateway.getProfile(email);
```

Comment:
```suggestion
       return gateway.getProfile(email);
```

Reply:
Creating a local object is not necessary. Keep the code clean and refactor things like this.